learn/guides/ballerina-shell.md

```diff
@@ -31,23 +34,39 @@ Type /exit to exit and /help to list available commands.
 =$ 
 ```
 
-## Stopping the Shell
+## Available commands
 
-Execute the `/exit` command to exit the Shell.
+Execute the `/help` command to view available commands.
```

Comment:
```suggestion
Execute the `/help` command to view the available commands.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75